src/solver/ts-generator/generator.h

```diff
@@ -34,12 +34,11 @@
 #include <antares/writer/i_writer.h>
 #include "xcast/xcast.h"
 
-namespace Antares
-{
-namespace Solver
-{
-namespace TSGenerator
+namespace Antares::Solver::TSGenerator
```

Comment:
```suggestion
namespace Antares::TSGenerator
```

Reply:
You suggest that namespace **TSGenerator** should not be contained in namespace  **Solver**.
That's right.
I moved  **TSGenerator** to **Antares** (many changes).